frontend/src/components/object/ObjectUpload.vue

```diff
@@ -67,6 +67,11 @@ const onRemoveUploadedFile = async (index: number) => {
 const onRemoveFailedFile = async (index: number) => {
   failedFiles.value.splice(index, 1);
 };
+
+// Based on files prop from upload component, are we in ready to upload mode
+const noFilesChosen = (files: File[]) => {
+  return !files || files.length === 0;
+};
```

Comment:
TS style - we're aiming to standardize array notations on angle bracket notation. Also the following would be equivalent leveraging null cascading syntax.
```ts
const noFilesChosen = (files?: Array<File>): boolean => !x?.length;
```

Reply:
fixed